parse-report.js

```diff
@@ -27,6 +27,11 @@ export default async function parseReport(source) {
     diagnosticMessage = ''
   }
 
+  // Removes file: form the start of the string
+  function parseFilePath(file) {
+    return file.replace(/^file:/, '')
+  }
```

Comment:
q: would it be safer to use the url module here instead of string replacement?

Reply:
Thanks @simoneb, I've done a safer implementation right now.